src/Controls/src/Core/Handlers/Items/iOS/TemplatedCell.cs

```diff
@@ -93,6 +94,24 @@ CGSize UpdateCellSize()
 			return size;
 		}
 
+		[Obsolete]
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		protected void Layout(CGSize constraints)
```

Comment:
@rmarinho you removed this in #7447 and it does not seem to be used. Not sure if adding back as it was is good or I should throw or something?

Reply:
no it wasn't used anywhere that's why i removed, but is fine to add it back